communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -1,9 +1,382 @@
+
+            (7) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/abricate/abricate" target="_blank">ABRicate</a>
+
+            (8) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/isescan/isescan" target="_blank">ISEScan</a>, 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/transit_gumbel/transit_gumbel" target="_blank">Transit</a>, 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/plasmidfinder/plasmidfinder" target="_blank">PlasmidFinder</a>
+
+            (9) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/staramr/staramr_search" target="_blank">StarAMR</a>, 
+            chewBBaca (e.g. <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca_createschema/chewbbaca_createschema" target="_blank">chewBBaca create schema</a>), 
+            pyMLST (e.g. <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/bvalot/pymlst/wgmlst_add_wrapper" target="_blank">Add strain to cg/wgMLST database</a>)), 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/mlst/mlst" target="_blank">MLST</a>, 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/nml/sistr_cmd/sistr_cmd" target="_blank">SISTR</a>  
+
+            (10) <a href="{{ galaxy_base_url }}/root?tool_id=https://toolshed.g2.bx.psu.edu/repos/dcouvin/crisprcasfinder" target="_blank">CRISPRCasFinder</a>  
+
+            (11) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/antismash/antismash" target="_blank">antiSMASH</a>, 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/hmmer_hmmsearch/hmmer_hmmsearch" target="_blank">HMMER</a> 
+
+            (12) Gene Ontology, 
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/kobas/kobas_identify" target="_blank">KEGG</a>
+            
+            (13) Orthofinder, 
+            OrthoMCL, 
+            Roary, 
+            Mauve, 
+            Clustal (Omega), 
+            FastTree, 
+            RAxML, 
+            Circos 
+            
+            (14) BWA, 
+            Bowtie, 
+            Minimap, 
+            Picard Tools 
+            
+            (15) FreeBayes, 
+            GATK, 
+            Bcftools, 
+            Snippy, 
+            medaka, 
+            Clair3 
+
+            (16) VCFtools. SnpSift, 
+            SnpEff, 
+            Samtools 
+
+            (17) STAR, 
+            HISAT, 
+            Bowtie2 
+
+            (18) featureCounts, 
+            HTSeq 
+
+            (19) edgeR, 
+            DESeq2 
+
+            (20) StringTie, 
+            GFFCompare, 
+            Cufflints 
```

Comment:
Please add the links there

Reply:
yeah this is the missing part I still work on mean while you review the other tabs of the section :)